.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .pytest_cache/
 **__pycache__/
 **db.sqlite3
+!/cinema.sqlite
```

Comment:
The negation rule `!/cinema.sqlite` suggests that you want to include `cinema.sqlite` in version control. Ensure that this is intentional, as database files are typically excluded from version control to avoid conflicts and maintain privacy.